### ISSUE TITLE
Supprimer la notion de "Itinéraire Lié" dans Geotrek-Admin ?

### ISSUE THREAD

**Issue by babastienne** [GeotrekCE/Geotrek-admin#3750]
J'ai l'impression que cette fonctionnalité n'est plus utilisée dans Geotrek-Rando V3. Est-ce qu'il faut la conserver ou bien on peut la retirer ? J'ai l'impression que ça peut contribuer à induire des utilisateurs en erreur lors de la saisie. 

ping @camillemonchicourt

**Comment by camillemonchicourt** [comment]
Oui, je l'avais indiqué dans un ticket que c'était confusant et plus utilisé, mais je ne retrouve pas le ticket en question. :-)

Mais oui à supprimer, ainsi que les données dans la BDD car peu utile ni pertinent.

**Comment by babastienne** [comment]
Lié à #3752

**Comment by camillemonchicourt** [comment]
Comme indiqué, ce champs n'était plus utilisé depuis un moment (depuis Geotrek-rando v3) et n'avait eu un usage ni un intérêt clair et identifié.

![image](https://github.com/GeotrekCE/Geotrek-admin/assets/4418840/21d0657e-8dbc-4015-8dc6-6df5f88b3636)

C'est pour cela qu'il a été supprimé et cela est effectif depuis la version 2.105.0.

La table et les données éventuelles qu'elle comportait sont donc supprimés automatiquement en passant à la version 2.105.0 ou plus.
